AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -333,6 +333,7 @@ forbes.com#@#.AD-label300x250
 forbes.com#@#.AD-Rotate
 !
 mail.yahoo.com##div[aria-describedby="adblock-whitelist-cue"]
+||prepareplanes.com^$third-party
```

Comment:
move close to anti-admiral rules

Reply:
somewhere under this rule?
`||courier-js.dallasnews.com/dist/courier.min.js`

Reply:
yep. Next time.